prepare-for-current-kernel.sh

```diff
@@ -7,8 +7,11 @@ then
 elif { echo $VERSION ; echo "5.99" ; } | sort -V -c 2>/dev/null
 then
   PATCHFILE="patch5.8"
-else
+elif { echo $VERSION ; echo "6.0" ; } | sort -V -c 2>/dev/null
```

Comment:
This actual means "VERSION <= 6.0" -> thus it won't match 6.01 etc...
Comparing against "6.19" would be better so it only uses the new patch from 6.2 onwards

Reply:
so, change the fllowing 
```
elif { echo $VERSION ; echo "6.0" ; } | sort -V -c 2>/dev/null
```
to 
```
elif { echo $VERSION ; echo "6.19" ; } | sort -V -c 2>/dev/null
```
right ?

Reply:
Yes exactly